### ISSUE TITLE
[QUESTION] finetuning with gradient accumulation doesn't currently work

### ISSUE THREAD

**Issue by BFeng14** [NVIDIA/Megatron-LM#418]
Hi, we found in tasks/finetune_utils.py that microbatches>1 is not supported in the finetune stage, and now we need to set microbatches>1 to get larger global_batch_size. Is there any hidden question behind that? 

```
def _train(model, optimizer, opt_param_scheduler, forward_step,
           train_dataloader, valid_dataloader, end_of_epoch_callback):
    """Train the model."""
    args = get_args()
    timers = get_timers()

    assert get_num_microbatches(
    ) == 1, "finetuning with gradient accumulation doesn't currently work"
```
